labels.yaml

```diff
@@ -0,0 +1,284 @@
+  - color: 0052cc
+    name: area/ingress
+  - color: 0052cc
+    name: area/introspection
+  - color: 0052cc
+    name: area/isolation
+  - color: 0052cc
+    name: area/kube-proxy
+  - color: 0052cc
+    name: area/kubeadm
+  - color: 0052cc
+    name: area/kubectl
+  - color: 0052cc
+    name: area/kubelet
+  - color: 0052cc
+    name: area/kubelet-api
+  - color: 0052cc
+    name: area/logging
+  - color: 0052cc
+    name: area/monitoring
+  - color: 0052cc
+    name: area/node-e2e
+  - color: 0052cc
+    name: area/node-lifecycle
+  - color: 0052cc
+    name: area/nodecontroller
+  - color: d4c5f9
+    name: area/os/coreos
+  - color: d4c5f9
+    name: area/os/fedora
+  - color: d4c5f9
+    name: area/os/gci
+  - color: d4c5f9
+    name: area/os/ubuntu
+  - color: d4c5f9
+    name: area/platform/aws
+  - color: d4c5f9
+    name: area/platform/azure
+  - color: d4c5f9
+    name: area/platform/gce
+  - color: d4c5f9
+    name: area/platform/gke
+  - color: d4c5f9
+    name: area/platform/mesos
+  - color: d4c5f9
+    name: area/platform/vagrant
+  - color: d4c5f9
+    name: area/platform/vsphere
+  - color: 0052cc
+    name: area/release-infra
+  - color: 0052cc
+    name: area/reliability
+  - color: 0052cc
+    name: area/rkt
+  - color: 0052cc
+    name: area/secret-api
+  - color: d93f0b
+    name: area/security
+  - color: 0052cc
+    name: area/stateful-apps
+  - color: 0052cc
+    name: area/swagger
+  - color: 0052cc
+    name: area/system-requirement
+  - color: 0052cc
+    name: area/teardown
+  - color: 0052cc
+    name: area/test
+  - color: 0052cc
+    name: area/test-infra
+  - color: 0052cc
+    name: area/third-party-resource
+  - color: 0052cc
+    name: area/ui
+  - color: 0052cc
+    name: area/upgrade
+  - color: 0052cc
+    name: area/usability
+  - color: 0052cc
+    name: area/workload-api/cronjob
+  - color: 0052cc
+    name: area/workload-api/daemonset
+  - color: 0052cc
+    name: area/workload-api/deployment
+  - color: 0052cc
+    name: area/workload-api/job
+  - color: 0052cc
+    name: area/workload-api/replicaset
+  - color: fef2c0
+    name: priority/awaiting-more-evidence
+  - color: fbca04
+    name: priority/backlog
+  - color: e11d21
+    name: priority/critical-urgent
+  - color: e11d21
+    name: priority/failing-test
+  - color: eb6420
+    name: priority/important-longterm
+  - color: eb6420
+    name: priority/important-soon
+  - color: ff0000
+    name: priority/P0
+  - color: ededed
+    name: priority/P1
+  - color: ededed
+    name: priority/P2
+  - color: ededed
+    name: priority/P3
+  - color: e11d21
+    name: 'cncf-cla: no'
+  - color: bfe5bf
+    name: 'cncf-cla: yes'
+  - color: 0ffa16
+    name: approved
+  - color: e11d21
+    name: do-not-merge
+  - color: ededed
+    name: needs-ok-to-merge
+  - color: b60205
+    name: needs-ok-to-test
+  - color: BDBDBD
+    name: needs-rebase
+  - color: fbca04
+    name: ok-to-merge
+  - color: ffaa00
+    name: queue/blocks-others
```

Comment:
The queue labels were created to replace the numerical priority labels for submit queue ordering.